storage/src/meta/mod.rs

```diff
@@ -55,6 +55,9 @@ pub mod toc;
 mod zran;
 pub use zran::{ZranContextGenerator, ZranInflateContext};
 
+mod batch;
+pub use batch::{BatchInflateContext, BatchInflateContextGenerator};
```

Comment:
Changed. But I'm still questioning whether this change is right, since `ZranContextGenerator` corresponds to `ZranContext` but `BatchContextGenerator` does not have a corresponding struct `BatchContext` currently.